xarray/coding/variables.py

```diff
@@ -212,11 +212,31 @@ class CFScaleOffsetCoder(VariableCoder):
         decode_values = encoded_values * scale_factor + add_offset
     """
 
+    @staticmethod
```

Comment:
This is a small nit, but I prefer not to use `staticmethod` when a normal function will do. We actually say "Avoid `@staticmethod`" in our internal Google style guide (which has gotten out of sync with the public one).